lib/lolapi.js

```diff
@@ -148,7 +148,14 @@
             if (options.endIndex) {
                 historyOptions += '&endIndex=' + options.endIndex;
             }
+            if (options.beginTime) {
```

Comment:
Can you make the check look for undefined? If someone used a 0 begin time for example this would return false.